components/esp_lvgl_port/src/lvgl9/esp_lvgl_port.c

```diff
@@ -159,10 +207,15 @@ static void lvgl_port_task(void *arg)
         } else if (task_delay_ms < 1) {
             task_delay_ms = 1;
         }
-        vTaskDelay(pdMS_TO_TICKS(task_delay_ms));
+
+        /* Sleep task, when everything done */
+        if (lv_anim_count_running() == 0 && lv_display_get_inactive_time(NULL) > 2 * LV_DEF_REFR_PERIOD) {
```

Comment:
From https://docs.lvgl.io/master/API/display/lv_display.html:
```
uint32_t lv_display_get_inactive_time(const lv_display_t *disp)
Get elapsed time since last user activity on a display (e.g. click)
Parameters: disp -- pointer to a display (NULL to get the overall smallest inactivity)
Returns: elapsed ticks (milliseconds) since the last activity
```